tests/foreman/cli/test_import.py

```diff
@@ -96,33 +98,42 @@ def update_csv_values(csv_file, new_data=None):
     result = csv_to_dataset(csv_file)
     for change in new_data:
         for record in result:
-            if record['organization_id'] == change['org_id']:
+            if record['organization_id'] == change['organization_id']:
                 record.update(change)
     return build_csv_file(result)
 
 
 def positive_import_org_data():
     return ([
-        {u'org_id': u'1', u'organization': gen_string('alphanumeric')},
-        {u'org_id': u'2', u'organization': gen_string('alphanumeric')},
-        {u'org_id': u'3', u'organization': gen_string('alphanumeric')},
+        {
+            u'organization_id': u'1',
+            u'organization': gen_string('alphanumeric')
+        },
+        {
+            u'organization_id': u'2',
+            u'organization': gen_string('alphanumeric')
+        },
+        {
+            u'organization_id': u'3',
+            u'organization': gen_string('alphanumeric')
+        },
     ],)
 
 
 def positive_import_user_data():
```

Comment:
@Ichimonji10:
done, the docstring is very simple as the purpose of the method is just to return a randomized data.
The same modest docstrings are used in `test_org` `*_data` methods